restaurant/settings.py

```diff
@@ -140,6 +141,9 @@
     'default': dj_database_url.config()
 }
 
+if 'test' in sys.argv:
+    DATABASES['default'] = {'ENGINE': 'django.db.backends.postgresql'}
```

Comment:
It's bad idea to check command line arguments from underlying modules as tests could be run in differ way than `./manage.py test`. I'd rather just remove this check and set default DB in all cases.